python/setup.py

```diff
@@ -135,7 +135,7 @@ def remove_if_exists(file_path):
         package_data=PACKAGE_DATA,
         license="https://www.apache.org/licenses/LICENSE-2.0",
         author="FeatHub Authors",
-        python_requires=">=3.6",
+        python_requires=">=3.7",
```

Comment:
Can we also require that the version <= 3.9?

Reply:
We can. But I find that PyFlink doesn't specify the upper limit of the Python version so I think we should not put a tighter constraint on Feathub. And we can add the classifiers to specify the Python version, like [PyFlink](https://github.com/apache/flink/blob/cc552f970e2673f2895ca3c73a8150cdf720915a/flink-python/setup.py#L337). What do you think?

Reply:
After thinking about this more, I agree it is better to skip specifying the version uppper-limit and follow the practice of pyflink/pyflink.